src/modules/global_css/index.js

```diff
@@ -37,6 +40,16 @@ class GlobalCSSModule {
     });
   }
 
+  setAutoTheme(value) {
```

Comment:
does the setting actually work? on initial loads this function does not check the value of the setting, meaning it will always add a listener and update the theme for browser color scheme on page load

Reply:
No it didn't work as it should. Since it's been awhile, i've just rewrote and reimplemented it for the most recent master.